tests/unit/records/test_schemas_datacite.py

```diff
@@ -97,7 +97,7 @@ def test_full(db, record_with_bucket, recid_pid):
                 "nameIdentifier": {
                     "nameIdentifier": "0000-0002-1825-0097",
                     "nameIdentifierScheme": "ORCID",
-                    "schemeURI": "http://orcid.org/"
+                    "schemeURI": "https://orcid.org/"
```

Comment:
Looks like the orcid changes need some more fixes in serializer: https://travis-ci.org/zenodo/zenodo/jobs/315233527#L6482